Automattic/delphin
# Add a list of hosts to which your site can connect

![screen shot 2016-06-21 at 4 18 06 pm](https://cloud.githubusercontent.com/assets/1130674/16244808/c396baf0-37cb-11e6-8579-f239496acd37.png)

![screen shot 2016-06-21 at 4 18 10 pm](https://cloud.githubusercontent.com/assets/1130674/16244811/c8155884-37cb-11e6-96b6-1affa6c0e4a4.png)

This PR:
- Adds `lib/hosts`, with data about available hosts.
- Adds `Hosts`, which displays a series of available hosts.
- Adds `HostInfo`, which displays information about a single host.
- Is large. ~150 lines are static data or from an SVG, but it is still big. If anyone prefers, I'm happy to move `HostsInfo` into a separate PR.

Still remaining is to add a 'cover image' to each host page, and make 'Connect' work.

**Testing**
- Visit `/hosts`
- Assert that you see a list of hosts.
- Click 'Learn More' ('Connect Now' does nothing).
- Assert that you are taken to information about this host.

@Automattic/sdev-feed
